src/components/structures/RoomView.tsx

```diff
@@ -608,6 +621,8 @@ export default class RoomView extends React.Component<IProps, IState> {
                 atEndOfLiveTimeline: this.messagePanel.isAtEndOfLiveTimeline(),
             });
         }
+
+        this.onResize();
```

Comment:
Are we calling this on every update? Won't this cause a loop? What am I missing? :sweat_smile:

Reply:
It doesn't create a loop. 🤷
It might make sense to split the width detection part out into another function and call it from `onResize()` and here though ...

Reply:
Yeah, you're right, it doesn't. I assume this is because of `shouldComponentUpdate()` which I didn't realize was there. You can mark this as resolved because GH doesn't let me do it...